documentation/ipi-install/modules/ipi-install-configuring-nodes.adoc

```diff
@@ -59,3 +59,20 @@ The installation process requires one NIC:
 NICx is a routable network (`baremetal`) that is used for the installation of the {product-title} cluster, and routable to the internet.
 
 endif::[]
+
+ifeval::[{release} > 4.6]
+.Configuring nodes for SecureBoot
+
+SecureBoot prevents a node from booting unless it verifies the node is using only trusted software, such as UEFI firmware drivers, EFI applications and the operating system. Red Hat only supports SecureBoot when deploying with RedFish Virtual Media.
+
+To enable SecureBoot, refer to the hardware guide for the node. You must perform the following:
+
+. Enable booting with UEFI. SecureBoot requires UEFI enabled.
+. Enable SecureBoot. Nodes usually disable SecureBoot by default.
+. Configure the key.
```

Comment:
I'd change this a bit too

Enabling SecureBoot requires the following:
. Set boot mode to UEFI enabled.
. Enable the SecureBoot options via the BIOS menu. (SecureBoot disabled by default)

Reply:
I purposely left out configure the key, as its not supported